### ISSUE TITLE
Bug: Slow Processing Speeds

### ISSUE THREAD

**Issue by Mennaruuk** [AngelNull/YouDownloader#78]
### Describe the Bug
Processing speeds are long.

### Steps to Reproduce
1. Input [this YouTube video URL](https://www.youtube.com/watch?v=3t6bLugtJkQ)
2. Click Download
3. Wait until options show up for video and audio qualities
4. Download in highest qualities
5. Wait for video to download and process

### Expected Behavior
Processing takes a really long time, even with short videos. I tried downloading a 40-minute video using command prompt, and it took about the same time as downloading a 37-second video with this app.

### Traceback (If Applicable)
A traceback from console can go here if there is one, which will help figure out what the problem is.

### Additional Context
Thanks for making this Electron app! Much handier than command prompt. Also [the trans rights site](https://www.trans-info.xyz/) is one more reason for people to star your repo!

**Comment by AngelNull** [comment]
Hey! Thanks for opening an issue. I'm aware of the performance issues and am currently working towards rewriting this entire application from the ground up to improve not only performance but the UI as well. Given this was my first electron app the codebase is all a little messy to work with. I can't give any big ETA to when a full rewrite will be finished, but know that it is on my to-do list!

**Comment by Mennaruuk** [comment]
Yeah, don't worry about it please! I just wanted to bring the issue to your attention, but I definitely understand this might just be a hobby thing, and your time is precious, so do with it as you please. I'm so thankful you considered my request!
